.travis.yml

```diff
@@ -6,5 +6,5 @@ install:
   - pip install -r requirements.txt
 # command to run tests
 script:
-  - pylint -d C0301,W0511 src/ test/
+  - pylint -d C0301,W0511,C0111,R0201,W0640 src/ test/

```

Comment:
I'm fine with the exceptions `C0111` and `R0201`, but I think `W0640` shouldn't be ignored for all the cases. 
The params passed to the lambda function might change when the function is actually invoked in case it was declared inside a loop. This thread explains better: https://stackoverflow.com/questions/938429/scope-of-lambda-functions-and-their-parameters/938493#938493

If the warning is a false positive in your case then you can add the linting exception to your scope instead of doing it globally.

Reply:
Replaced lambda function with list comprehension, thanks for the suggestion @harshidw !